pelican/plugins/seo/tests/conftest.py

```diff
@@ -193,6 +193,52 @@ def fake_article_multiple_elements():
     )
 
 
+@pytest.fixture()
```

Comment:
You don't have to create a new fixture for your need. Just update https://github.com/pelican-plugins/seo/blob/master/pelican/plugins/seo/tests/conftest.py#L61 by adding the HTML code you need. Of course you'll need to update the test which use this fixture too.

Reply:
Removed the new test fixture and added additional content to the `fake_article` fixture.